resources/sdk/clisdkclient/extensions/utils/general.go

```diff
@@ -305,3 +308,14 @@ func MilliSecondsToNanoSeconds(milliSeconds int64) time.Duration {
 func SecondsToNanoSeconds(seconds int) time.Duration {
 	return MilliSecondsToNanoSeconds(int64(seconds)) * 1000
 }
+
+func ConvertToJSON(data string) string {
+	if data_format.InputFormat == "json" {
```

Comment:
Would be nice to do a case insensitive comparison to be more forgiving to the user. https://stackoverflow.com/a/67818307 would be a nice way of comparing any time you're checking for "json" or "yaml" with this feature